docs/detections/alerts-view-details.asciidoc

```diff
@@ -58,18 +58,13 @@ The Insights section provides the following details:
 
 * *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
 * *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
-* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session.
+* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session. To use this feature, the *Include session data* setting must be enabled on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

Comment:
Revising passive voice. Also, "Session ID" prob doesn't need to be capitalized; I don't think it's a proper name, just the ID for a given session.
```suggestion
* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```